go.mod

```diff
@@ -0,0 +1,43 @@
+module github.com/operator-framework/api
+
+go 1.12
+
+replace (
+	// Pin openshift version to 4.2 (uses kube 1.14)
+	github.com/openshift/api => github.com/openshift/api v3.9.1-0.20190717200738-0390d1e77d64+incompatible
+	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20190627172412-c44a8b61b9f4
+
+	// Pin kube version to 1.14
```

Comment:
We can bump to kube 1.16 now since OLM has already moved forward.

Reply:
Can we do this in a follow-up PR?